TrackingTools/KalmanUpdators/interface/Chi2MeasurementEstimatorBase.h

```diff
@@ -18,12 +18,13 @@ class Chi2MeasurementEstimatorBase : public MeasurementEstimator {
    *  The errors of the trajectory state are multiplied by nSigma 
    *  to define acceptance of Plane and maximalLocalDisplacement.
    */
-  explicit Chi2MeasurementEstimatorBase(double maxChi2, double nSigma = 3.) : 
-    theMaxChi2(maxChi2), theNSigma(nSigma) {}
+  explicit Chi2MeasurementEstimatorBase(double maxChi2, double nSigma = 3., float maxDisp=100.) : 
```

Comment:
it looks like maxdisp=100 is degrading the cosmic muon reconstruction (the standalone 1-leg muons are the most affected, cosmicMuons1Leg track collection).
https://github.com/cms-sw/cmssw/blob/CMSSW_8_0_X/RecoMuon/TrackingTools/src/MuonTrajectoryUpdator.cc 
MuonTrajectoryUpdator instantiates Chi2MeasurementEstimator directly.
I suppose arguments with a larger value should be passed.

This is just a guess.